source/Nevermore/Advanced/ReadTransaction.cs

```diff
@@ -642,14 +642,14 @@ protected async Task<TResult[]> ReadResultsAsync<TResult>(PreparedCommand prepar
             if (mapping.IdColumn?.Type != typeof(TKey))
                 throw new ArgumentException($"Provided Id of type '{typeof(TKey).FullName}' does not match configured type of '{mapping.IdColumn?.Type.FullName}'.");
 
-            var columnNames = GetColumnNames(mapping.SchemaName, mapping.TableName);
+            var columnNames = GetColumnNames(configuration.GetSchemaNameOrDefault(mapping), mapping.TableName);
```

Comment:
Since it's used further down, I think it's worth pulling into a local variable `schema`

Reply:
Good spot! I didn't even see that in the SQL below.